frontend/src/js/components/settings/SettingsPage.tsx

```diff
@@ -1,24 +1,33 @@
-import React, {Fragment} from 'react';
+import React, {useState, Fragment} from 'react';
 import Paper from '@mui/material/Paper';
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
 
 import SettingsEditor from '../../containers/SettingsEditor';
 import CurrencyEditor from '../../containers/CurrencyEditor';
 import CategoryViewer from '../../containers/CategoryViewer';
 
 export function SettingsPage() {
-        return (
-            <Fragment>
-                <Paper variant='outlined'>
-                    <SettingsEditor/>
-                </Paper>
-                <Paper variant='outlined'>
-                    <CurrencyEditor/>
-                </Paper>
-                <Paper variant='outlined'>
-                    <CategoryViewer/>
-                </Paper>
-            </Fragment>
-        )
+    const [tabValue, setTabValue] = useState('settings');
+
+    const switchTab = (ev, value) => {
+        setTabValue(value);
+    }
```

Comment:
```suggestion

```

Reply:
I disagree with the suggestion as the initial solution is more explicit and hence clear.